source/funkin/play/PauseSubState.hx

```diff
@@ -311,7 +316,20 @@ class PauseSubState extends MusicBeatSubState
     metadataDifficulty.scrollFactor.set(0, 0);
     metadata.add(metadataDifficulty);
 
-    metadataDeaths = new FlxText(20, metadataDifficulty.y + 32, FlxG.width - 40, '${PlayState.instance?.deathCounter} Blue Balls');
+    metadataDeaths = new FlxText(20, metadataDifficulty.y + 32, FlxG.width - 40, '');
+    if (Preferences.naughtyness) {
+      if PlayState.instance?.deathCounter = 1 {
+        metadataDeaths = new FlxText(20, metadataDifficulty.y + 32, FlxG.width - 40, '${PlayState.instance?.deathCounter} Blue Ball');
+      } else {
+        metadataDeaths = new FlxText(20, metadataDifficulty.y + 32, FlxG.width - 40, '${PlayState.instance?.deathCounter} Blue Balls');
+      }
+    } else {
+      if PlayState.instance?.deathCounter = 1 {
+        metadataDeaths = new FlxText(20, metadataDifficulty.y + 32, FlxG.width - 40, '${PlayState.instance?.deathCounter} Death');
+      } else {
+        metadataDeaths = new FlxText(20, metadataDifficulty.y + 32, FlxG.width - 40, '${PlayState.instance?.deathCounter} Deaths');
+      }
+    }
```

Comment:
This entire section can be compacted into
```
metadataDeaths = new FlxText(20, metadataDifficulty.y + 32, FlxG.width - 40, '${PlayState.instance?.deathCounter} ${Preferences.naughtyness ? 'Blue Ball' : 'Death'}${PlayState.instance?.deathCounter == 1 ? '' : 's'}');
```
It's a bit long though so maybe you might wanna convert certain things into variables
`${Preferences.naughtyness ? 'Blue Ball' : 'Death'}` for example could be
```
var deathText = Preferences.naughtyness ? 'Blue Ball' : 'Death';
metadataDeaths = new FlxText(20, metadataDifficulty.y + 32, FlxG.width - 40, '${PlayState.instance?.deathCounter} $deathText${PlayState.instance?.deathCounter == 1 ? '' : 's'}');
```
And if you really wanted to go further you could do convert the s logic as well
```
var deathText = Preferences.naughtyness ? 'Blue Ball' : 'Death';
var possiblyS = PlayState.instance?.deathCounter == 1 ? '' : 's';
metadataDeaths = new FlxText(20, metadataDifficulty.y + 32, FlxG.width - 40, '${PlayState.instance?.deathCounter} $deathText$possiblyS');
```
Up to you

In case you don't quite understand what's going on, there's a thing in many programming languages called ternary operators and they're basically an if else statement in a single line, the first section being the condition, the thing after the ? being the if, and the thing after the : being the else

Also for future reference you can just do `textThing.text = 'blah';` instead of making an entirely new FlxText
These are all just suggestions, I don't wanna tell you how to program, but they're nice things to know

Reply:
Thank you! I am new-ish to HaxeFlixel so this will help a ton! :)